### ISSUE TITLE
SupportBundle always saves to $HOME from the results screen

### ISSUE THREAD

**Issue by adamancini** [replicatedhq/troubleshoot#697]
**Bug Description**

When running preflights from the CLI, at the results screen at the end, the user hits `s` to save and the results file is written to $CWD.  When running supportbundle from the CLI, however, it seems the results file is written to $HOME instead.  

**Expected Behavior**

The interfaces should be consistent in their behaviour, which should probably be to emit to $CWD.

**Steps To Reproduce**

run a minimal SupportBundle spec such as 

```
apiVersion: troubleshoot.sh/v1beta2
kind: SupportBundle
metadata:
  name: ping
spec:
  collectors:
    - runPod:
        name: "run-ping"
        namespace: default
        podSpec: 
          containers:
          - name: run-ping
            image: busybox:1
            command: ["sh"]
            args: ['-c', 'echo SMTP server is reachable , PASS , blah blah message']
            imagePullPolicy: IfNotPresent
  analyzers:
    - textAnalyze:
        checkName: "run-ping"
        fileName: run-ping/run-ping.log
        regexGroups: 'SMTP server is reachable , (?P<Result>.*?) , (?P<Message>.*$)'
        outcomes:
          - pass:
              when: "Result == PASS"
              message: Passed
          - fail:
              when: "Result == FAIL"
              message: 'Result: {{ .Result }}'
```

from the CLI, and at the results screen, hit `s` to save results.  Observe the file gets written to `$HOME/support-bundle-results.txt`

**Additional Context**
```
~/.krew/bin/kubectl-support_bundle version
Replicated Troubleshoot 0.40.0
```

**Comment by adamancini** [comment]
<img width="1224" alt="Screen Shot 2022-09-01 at 14 46 26" src="https://user-images.githubusercontent.com/292598/187999453-e13ce30f-a55d-4447-b7de-16c359436162.png">

**Comment by adamancini** [comment]
https://github.com/replicatedhq/troubleshoot/pull/532

**Comment by xavpaice** [comment]
Closed with #538
